test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -322,10 +322,10 @@ export default class PerfCompare extends Component {
                 testBuildURL = testURLScheme + "//" + testHostWithPort;
 
                 if (testBuildNum === undefined || (testBuildURL === undefined )) {
-                    displayErrorMessage += "Invalid Test URL. "
+                    displayErrorMessage += "Error: Invalid Test URL. "
                 }
             } catch (testBuildURLSplit) {
-                displayErrorMessage += "Invalid Test URL. "
+                displayErrorMessage += "Please type in a test URL. "
```

Comment:
Don't think this fixes the problem fully. I think we will still get a red notification, but with a different message now. We shouldn't get any notification at all when you load Perf Compare.